tidy3d/material_library/material_library.py

```diff
@@ -185,17 +188,21 @@ class MaterialItem2D(MaterialItem):
 
 Ag_Yang2015Drude = VariantItem(
```

Comment:
Similarly, the original parameters seem to come from Drude model directly. Could you or @tomflexcompute who have access to the paper check if the paper provides a Drude model?

Reply:
Yes these poles are directly converted from the Drude model with the parameter values taken from the paper directly.